spyder/app/utils.py

```diff
@@ -112,13 +115,9 @@ def setup_logging(cli_options):
         console_filters = [x for x in console_filters if x != '']
 
         handlers = [logging.StreamHandler()]
-        if cli_options.debug_output == 'file':
-            log_file = 'spyder-debug.log'
-            handlers.append(
-                logging.FileHandler(filename=log_file, mode='w+')
-            )
-        else:
-            log_file = None
+        handlers.append(
+            logging.FileHandler(filename=SPYDER_DEBUG_LOG_FILE, mode='w+')
+        )
```

Comment:
Please make this work for our command line interface and using a `Restart Spyder in debug mode` action in the Tools menu.

Reply:
will do